eval.go

```diff
@@ -76,13 +76,18 @@ func (e *SetExpr) eval(app *App, args []string) {
 	case "shell":
 		gOpts.shell = e.val
 	case "showinfo":
-		if e.val != "none" && e.val != "size" && e.val != "time" {
-			msg := "showinfo should either be 'none', 'size' or 'time'"
+		if e.val != "none" && e.val != "size" && e.val != "time" && e.val != "time:size" && e.val != "size:time" {
+			msg := "showinfo should either be 'none', 'size', 'time', 'time:size' or 'size:time'"
 			app.ui.message = msg
 			log.Print(msg)
 			return
 		}
-		gOpts.showinfo = e.val
+		var tmp []string
+		tmp = strings.Split(e.val, ":")
```

Comment:
@IvanMenshykov why do you split to `tmp` variable and then add it to `gOpts.showinfo` one by one instead of just `gOpts.showinfo = strings.Split(e.val, ":")`?